tests/providers/snowflake/operators/test_snowflake_to_slack.py

```diff
@@ -0,0 +1,100 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from unittest.mock import Mock
+
+import pytest
+
+from airflow.models import DAG
+from airflow.providers.snowflake.operators.snowflake_to_slack import (
+    SlackWebhookHook, SnowflakeHook, SnowflakeToSlackOperator,
+)
+from airflow.utils import timezone
+
+TEST_DAG_ID = 'snowflake_to_slack_unit_test'
+DEFAULT_DATE = timezone.datetime(2017, 1, 1)
+
+
+@pytest.fixture
+def mock_snowflake_hook_class(monkeypatch):
+    mocked_hook = Mock(spec=SnowflakeHook)
+    monkeypatch.setattr('airflow.providers.snowflake.operators.snowflake_to_slack.SnowflakeHook', mocked_hook)
+    return mocked_hook
+
+
+@pytest.fixture
+def mock_slack_hook_class(monkeypatch):
+    mocked_hook = Mock(spec=SlackWebhookHook)
+    monkeypatch.setattr('airflow.providers.snowflake.operators.snowflake_to_slack.SlackWebhookHook',
+                        mocked_hook)
+    return mocked_hook
+
+
+@pytest.fixture
+def mock_example_dag():
```

Comment:
```suggestion
def example_dag():
```
Why is it prefixed with `mock_`? It does not mock anything, does it?